hooks/post-checkout

```diff
@@ -29,9 +29,11 @@ function merge() {
 
   git fetch -v origin "${target_branch}"
   git checkout FETCH_HEAD
-  GIT_COMMITTER_EMAIL="auto-merge@buildkite" \
+  # env vars to ensure merge is non-interactive
   GIT_AUTHOR_NAME="github-merged-pr-buildkite-plugin" \
-  GIT_COMMITTER_NAME="github-merged-pr-buildkite-plugin" \
+  GIT_COMMITTER_EMAIL="auto-merge@buildkite" \
+  GIT_COMMITTER_NAME="github-merged-pr-buildkite-plugin"
+  GIT_MERGE_AUTOEDIT="no" \
   git merge --no-edit "${BUILDKITE_COMMIT}"
```

Comment:
Does this mean you don't need `--no-edit` on the next line?

Reply:
Can you please remove one of these? Per https://git-scm.com/docs/git-merge#git-merge---no-edit only `--no-edit` or `GIT_MERGE_AUTOEDIT` is required -- preference would be to remove the `--no-edit` flag

Happy to merge after that

Reply:
Sorry just saw this, have been _really_ busy. Committed 👍